code/controllers/subsystem/strings.dm

```diff
@@ -0,0 +1,10 @@
+SUBSYSTEM_DEF(strings)
+	name = "Strings"
+	init_order = INIT_ORDER_STRINGS
+	flags = SS_NO_FIRE
+
+/datum/controller/subsystem/strings/Initialize(timeofday)
+	for(var/i in GLOB.string_files)
+		GLOB.vars[i] = file2list(GLOB.string_files[i])
```

Comment:
If a file is large for whatever reason this could lock up? 
Could you add the TICK_CHECK here?

Reply:
it's init so i dont think it matters